scripts/vtam-enum.nse

```diff
@@ -200,10 +205,12 @@ local function vtam_test( host, port, commands, macros)
   tn:get_all_data()
   tn:get_screen_debug(2)
   local isVTAM = false
-  if tn:find('IBMECHO ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789') then
+  if tn:find('IBMECHO ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789') and not macros then
```

Comment:
Very minor nitpick, but test `macros` first before the more-expensive string find operation. Lua short-circuits logic just like C.